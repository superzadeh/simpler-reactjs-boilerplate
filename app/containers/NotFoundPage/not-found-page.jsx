/**
 * NotFoundPage
 *
 * This is the page we show when the user visits a url that doesn't have a route
 */

import React from 'react';

export const NotFoundPage = () => (
  <div>Not found :(</div>
);

export default NotFoundPage;

