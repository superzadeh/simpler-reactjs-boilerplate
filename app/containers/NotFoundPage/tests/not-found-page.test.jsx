import React from 'react';
import { shallow } from 'enzyme';

import { NotFoundPage } from '../not-found-page';

const getElement = () => (<NotFoundPage></NotFoundPage>);

describe('NotFoundPage', () => {
  it('should render correctly', () => {
    const element = getElement();
    expect(element).toMatchSnapshot();
  });
  it('should render as a <div> tag', () => {
    const renderedComponent = shallow(getElement());
    expect(renderedComponent.type()).toEqual('div');
  });
});
