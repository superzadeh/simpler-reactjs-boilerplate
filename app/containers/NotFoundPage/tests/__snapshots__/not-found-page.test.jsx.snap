// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render correctly 1`] = `<Unknown />`;
